src/Websites/Websites/help/Publish-AzWebApp.md

```diff
@@ -197,6 +199,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -ProgressAction
```

Comment:
I think ProgressAction is removed on "main" branch. Was this added back intentionally or just need to rebase your branch on main.